Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "memcache"
-version = "0.17.2"
+version = "0.18.0"
```

Comment:
Usually we don't bump the version in common changes, we just change it before release.